robottelo/ui/base.py

```diff
@@ -47,41 +49,39 @@ def find_element(self, locator):
                 return _webelement
             else:
                 return None
-        except NoSuchElementException as e:
-            logging.debug("%s: Could not locate element %s.",
-                          type(e).__name__,
-                          locator[1])
-            return None
-        except Exception, error:
-            logging.debug("Failed to locate element. ERROR: %s", str(error))
+        except NoSuchElementException as err:
```

Comment:
No, this is the selenium exception and not the robottelo one. It is checking for the selenium exception in order to return `None` when the element is not found.